package.json

```diff
@@ -101,7 +101,7 @@
     "coverage": "yarn run jest --coverage",
     "extract-strings": "./utils/extract-strings.js",
     "flow:ci": "flow check --max-workers 0",
-    "lint": "eslint --config .eslintrc.js --ext .js --ext .jsx",
+    "lint": "eslint --config .eslintrc.js . --ext .js --ext .jsx",
```

Comment:
😅 well that sure doesn't follow the [Principle of Least Astonishment](https://en.wikipedia.org/wiki/Principle_of_least_astonishment)!

Thanks for fixing this!